go/staking/api/api.go

```diff
@@ -1183,6 +1183,13 @@ type Genesis struct {
 
 // ConsensusParameters are the staking consensus parameters.
 type ConsensusParameters struct { // nolint: maligned
+	// TokenSymbol is the token's ticker symbol.
+	// Only upper case A-Z characters are allowed.
```

Comment:
Where is this enforced?

Reply:
It is not, but it cannot really be changed outside an upgrade handler.